### PR TITLE
Update harvard-stellenbosch-university.csl

### PR DESCRIPTION
**Pull request by Yusufras**
<text macro="author-short"/>
        <text macro="year-date"/>

### CODE REVIEW

File: harvard-stellenbosch-university.csl [hunk 1]
```diff
@@ -203,7 +203,7 @@
     <layout prefix="(" suffix=")" delimiter="; ">
       <group delimiter=", ">
         <text macro="author-short"/>
-        <text macro="date-issued-no-parentheses"/>
+        <text macro="year-date"/>
```
Comment (POBrien333) [inline harvard-stellenbosch-university.csl]: You've now called a macro (year-date) that does not exist and therefor it is giving you an error.
What are you trying to accomplish?

**Comment by Yusufras** [comment]
In my in text citation,

I want multiple authors to appear alphabetically instead of,

chronological as it is currently



Vriendelike groete / Kind regards
Yusuf Ras

Hoof: Geneeskunde en Gesondheidswetenskappe Biblioteek / Head: Medicine and Health Sciences Library
Biblioteek- & Inligtingsdiens / Library & Information Service
Universiteit Stellenbosch University  | map<https://www.google.co.za/maps/place/JS+Gericke+Library/@-33.932975,18.8640494,377m/data=!3m1!1e3!4m2!3m1!1s0x1dcdb25d82940c29:0x80f6dcd585958374> | Tel: +27 (021) 938 9365 | yr@sun.ac.za<mailto:yr@sun.ac.za>

**Comment by POBrien333** [comment]
@Yusufras 
You can just switch lines 200 and 201 with each other. 
Structure wise your in the "citation" part and in that in the "sort" part of the style. Just for your orientation :)

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**